### DISCUSSION TITLE
[Preprocessor] inpaint_only is added

### DISCUSSION THREAD

**Discussion by lllyasviel** [Mikubill/sd-webui-controlnet#1442] [General]
inpaint_only is a simple inpaint preprocessor that allows you to inpaint without changing unmasked areas (even in txt2img).

**Comment by ghost** [comment]
Is there any example workflow on how this feature works?

**Comment by herval** [comment]
how does it differ from inpaint_global_harmonious?

**Comment by lllyasviel** [comment]
inpaint_only never change unmasked areas (even in t2i) but inpaint_global_harmonious will change unmasked areas (without the help of a1111's i2i inpaint) 
If you use a1111's i2i inpaint, they are same.

**Comment by zmdzyy123-dot** [comment]
inpaint is well
